### ISSUE TITLE
Audio icon overlaps with tab logo icon

### ISSUE THREAD

**Issue by quissberry** [zen-browser/desktop#41]
If you are playing something in a tab, then you hover over the tab icon, the audio icon appears directly above the tab icon. Although the hit box of the audio icon is smaller than the tab icon, the hit boxes are not exactly clear, and I think it is easy to misclick especially if you are not used to it. It would be more ideal and would look more nice if the two icons were separated somehow (I am thinking directly below).  

![image](https://github.com/user-attachments/assets/5ee4057d-ead4-40ef-84fa-4ed4f931be9b)

**System information:**
Arch Linux, niri
Zen Browser Version 1.0.0-a.1 (64-bit)

**Comment by mauro-balades** [comment]
Thanks a lot for this feedback! What would be better?

- completly hide this mute icon
- open some sort of mini menu at the bottom with the mute icon?

**Comment by quissberry** [comment]
Honestly, I rarely use the mute audio so I wouldn't mind hiding it.

**Comment by mauro-balades** [comment]
will fix ASAP for the next release 👍🏽

**Comment by DaitiDay** [comment]
Hi, I've installed zen-browser today and I'm loving it. The sidebar is genius, fantastic feature. GREAT PROJECT!
Only thing is the aforementioned audio/muted icon. I often use the audio icon to identify which tab is producing sound, and just as often, I mute a tab by pressing the audio icon without even opening the tab. With the most recent update, I noticed that the icon has been completely removed, leaving me with the only options of opening the tab to pause the audio source or right-clicking to mute the tab. Is it possible to find an alternative solution? For example, in Firefox 'vanilla,' I used an extension to get a vertical tab bar similar to the one Zen has by default, and in that case, the audio and mute icons would appear this way.
![Screenshot 2024-08-20 180704](https://github.com/user-attachments/assets/b055c96c-3c22-4a78-ba99-a63cf7ef5189)
It was enough to press the icon to toggle between audio and mute. Is a similar solution implementable in Zen?
